bloptools/tests/test_bayesian_shadow.py

```diff
@@ -1,10 +1,12 @@
 import numpy as np
+import pytest
 from sirepo_bluesky.sirepo_ophyd import create_classes
 
 from bloptools.bo import BayesianOptimizationAgent
 from bloptools.experiments.shadow import tes
 
 
+@pytest.mark.shadow
```

Comment:
Are those markers used anywhere?

Reply:
I sometimes use them to exclude sirepo tests, I can split up the tests into sirepo and non-sirepo

Reply:
I remember the markers need to be declared in the `pytest.ini` file at the top of the repo.

Example 1: https://github.com/NSLS-II/sirepo-bluesky/blob/main/pytest.ini.
Example 2: https://github.com/Synchrotron-Radiation-Workshop/SRW/blob/reorg/clients/python/pytest.ini